backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/queries.html.ftl

```diff
@@ -0,0 +1,322 @@
+                queries = data;
+                updateQueriesTable();
+            })
+            .catch(error => {
+                console.log(error);
+            });
+
+    } catch (error) {
+        console.log(error);
+        return null;
+    };
+
+}
+<#noparse >
+    function getHtmltemplate(data) {
+        return `
+
+
+<tr>
+   <td >
+<div class="card container">
+  <div class="card-body">
+      <div class="row">
+         <div class="col title">
+            ${data.label} ( ${data.ownerName} , ${data.id.split('.')[0]} ) : ${data.id.split('.')[1]}
+         </div>
+      </div>
+
+
+
+      <div class="row container">
+         <div class="col inner">
+            Creation-Time : ${((new Date(data.createdAt)).toLocaleString("de-De"))}
+         </div>
+         <div class="col inner">
+            Start-Time : ${((new Date(data.startTime)).toLocaleString("de-De"))}
+         </div>
+         <div class="col inner">
+            Finish-Time : ${((new Date(data.finishTime)).toLocaleString("de-De"))}
+         </div>
+      </div>
+      <div class="row container">
+
+         <div class="col inner">
+            Requested-Time : ${data.requiredTime} ms
+         </div>
+           <div class="col inner">
+                     Query-Type : ${data.queryType}
+          </div>
+
+        <div class="col">
+           <button class="btn btn-primary innerBtn" type="button" data-toggle="collapse" data-target="#query${queryCounter}" aria-expanded="false" aria-controls="query${queryCounter}" ${(data.query ? '' : 'disabled')}>
+           Show query content
+           </button>
+
+           <button class="btn btn-primary innerBtn" type="button" data-toggle="collapse" data-target="#error${queryCounter}" aria-expanded="false" aria-controls="error${queryCounter}" ${(data.error ? '' : 'disabled')}>
+           Show errors
+           </button>
+        </div>
+      </div>
+
+
+
+      <div class="row container">
+        <div class="col">
+             <div class="progress">
+                <div class="progress-bar ${(data.status === "RUNNING" ? "bg-warning" : (data.status === "FAILED" ? "bg-danger" : (data.status === "DONE" ? "bg-success" : "") ))}" role="progressbar" style="width: ${(data.progress+k)}%" aria-valuenow="${(data.progress+k)}" aria-valuemin="0" aria-valuemax="100">
+                    ${(data.progress+k)} %
+                </div>
+             </div>
+        </div>
+      </div>
+
+
+		<div class="row container">
+			 <div class="col">
+				<div class="collapse multi-collapse" id="query${queryCounter}">
+				   <div class="card card-body">
+					 <pre> ${(data.query ? JSON.stringify(data.query, undefined, 2) : '')}  </pre>
+				   </div>
+				</div>
+			 </div>
+			 <div class="col">
+				<div class="collapse multi-collapse" id="error${queryCounter}">
+				   <div class="card card-body">
+					  <pre>  ${(data.error ? JSON.stringify(data.error, undefined, 2) : '')}  </pre>
```

Comment:
Ich weiß, nervig aber kannst du versuchen den Trace etwas zu rekonstruieren statt das rohe json zu printen?

Denke, die Message und dann `$Datei $Klasse::$Methode#$Zeile`  untereinander wäre ein guter Anfang?

Reply:
☝️

Reply:
Ich finde das in Ordnung, wie es gerade ist, denn ein Stacktrace ist zu verbose.